docs/release-notes.md

```diff
@@ -7,6 +7,7 @@ nav_order: 8
 ## Upcoming coreos-installer 0.17.0 (unreleased)
 
 Major changes:
+- install: if using `--save-partindex` add support for MBR device 
```

Comment:
Wording suggestion: Add support for MBR partition tables to `--save-partindex`

Reply:
waaay better, thank you